pkg/bloombuild/planner/strategies/chunksize.go

```diff
@@ -21,22 +22,47 @@ import (
 	"github.com/grafana/loki/v3/pkg/storage/stores/shipper/indexshipper/tsdb/index"
 )
 
+const (
+	metricsNamespace = "loki"
+	metricsSubsystem = "bloomplanner"
+)
+
+type ChunkSizeStrategyMetrics struct {
+	tenantTaskSize *prometheus.HistogramVec
+}
+
+func NewChunkSizeStrategyMetrics(reg prometheus.Registerer) *ChunkSizeStrategyMetrics {
+	return &ChunkSizeStrategyMetrics{
+		tenantTaskSize: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
```

Comment:
nit

```suggestion
			Namespace: constants.Loki,
```

Reply:
Cool! Let me do that on a followup PR so we applied them to all the metrics